### PR TITLE
Documentation for Shelly integration

### PR DESCRIPTION
**Pull request by OnFreund**
## Proposed change

Documentation for the new Shelly integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/36531
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/shelly.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+This integration can be configured using the integrations panel in the
+Home Assistant frontend.
+
+Menu: **Configuration** -> **Integrations**.
+
+Click on the `+` sign to add an integration and click on **Shelly**.
+The configuration flow will discover devices that are currently connected
+to your MQTT server, and will guide you through configuring them.
+
+Note: This integration is still work-in-progress, and currently only supports
```
Comment (frenck) [inline source/_integrations/shelly.markdown]: This note should be in here.

Please write what is support. Not what is currently in development. Also, avoid making promises like "Wider coverage is coming soon".

**Review by frenck — CHANGES_REQUESTED** [review]
One small comment @OnFreund!

Could you also make sure a PR @ our Brands repository is opened for this new integration?

Thanks 👍

**Comment by OnFreund** [comment]
Thanks @frenck. The brands repository already has the images (I didn't put them there, though, it was there before I started working on this).

**Comment by frenck** [comment]
@OnFreund It is in the custom_components folder, as this is a core integration, it needs to be moved. 
Please open a PR for that.

**Pull request closed without merging**